samples/snippets/sample_pagination.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START compute_images_list_page ]
+# [START compute_images_list ]
+import google.cloud.compute_v1 as compute_v1
+# [END compute_images_list ]
+# [END compute_images_list_page ]
+
+
+# [START compute_images_list ]
+def print_images_list(project: str) -> None:
+    """
+    Prints a list of all non-deprecated image names available in given project.
```

Comment:
Should this specify as "custom image names"?

ref: http://cloud.google.com/compute/docs/reference/rest/v1/images/list

Reply:
This method can be used to get both the "custom images" and the "public images". You just need to provide it with proper project name:

> If you want to get a list of publicly-available images, use this method to make a request to the respective image project, such as debian-cloud or windows-cloud.

I decided to list public images from the `windows-sql-cloud` project, because it doesn't require any setting up and doesn't generate any costs to anyone running this sample.